auth_ldaps/models/res_company_ldap.py

```diff
@@ -0,0 +1,50 @@
+# Copyright (C) Creu Blanca
+# Copyright (C) 2018 Brainbean Apps
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+
+import logging
+
+import ldap
+
+from odoo import fields, models
+
+_logger = logging.getLogger(__name__)
+
+
+class CompanyLDAP(models.Model):
+    _inherit = "res.company.ldap"
+    _description = "Company LDAP configuration"
```

Comment:
Not needed as it's an `_inherit`.

Reply:
Agreed that it's better not to do this, but this was also present in 16.0, 15.0, etc.

Maybe it's better to fix it after upgrade with a "fix" PR, so the fix can be backported.